packages/money-legos/maker.ts

```diff
@@ -1,70 +1,77 @@
 import erc20 from "./erc20";
 
-import networkIds from "./networks"
+import networkIds from "./networks";
 
 import makerProxyRegistryAbi from "./abi/makerdao/ProxyRegistry.json";
 import dssCdpManagerAbi from "./abi/makerdao/DssCdpManager.json";
 import dssProxyActionsAbi from "./abi/makerdao/DssProxyActions.json";
 
 const ilks = {
-  "BAT-A": {
+  bat_a: {
```

Comment:
how come underscore? why not `ethA`? What is the `A` for anyway?

Reply:
Im just following the makerdao std https://changelog.makerdao.com/releases/mainnet/1.0.4/contracts.json

Reply:
Those are constants exposed via a JSON API. For Javascript libraries, camelCase is always the convention. The only exception is if they are global constants, then you can use all-caps with underscores.

Source: https://www.crockford.com/code.html